src/Nav.php

```diff
@@ -81,11 +81,6 @@ class Nav extends Widget
      * @see isItemActive
      */
     public $activateItems = true;
-    /**
-     * @var bool whether to activate the container when the menu item is active.
-     * @since 2.0.3
-     */
-    public $activateContainer = true;
```

Comment:
That could be considered a backwards compatibility break. Is this property useless? If not, I'd like to keep it. If it is useless, we need to at least mention it in upgrade notes (by creating UPGRADE.md like the one in the core framework).

Reply:
It has become useless. I will check UPGRADE.md later today

Reply:
Do I place the file in src, or in the root folder?

Reply:
root.